<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="BotaoConecta">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Conecta</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nome">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="cargo">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ponto Eletronico</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Nome</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Cargo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Matrícula</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ultima Entrada</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ultima Saída</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="matricula">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="hora_entrada">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="hora_saida">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>200</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="data_entrada">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>170</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="data_saida">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="botaoLeDados">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Le Dados</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botaoApagaDados">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Apaga Dados</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botaoAtiva">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Envia/Ativa</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botaoCadastra">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cadastra</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>40</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Foto</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>512</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
